src/Styling/Navbar.css

```diff
@@ -0,0 +1,55 @@
+.Navbar {
+   width: 100vw;
+   height: 200px;
+   margin-top: 10px;
+   position: fixed;
+   display: flex;
+   justify-content: space-between;
+   cursor: pointer;
```

Comment:
Wouldn’t this cause the cursor to be pointer even in the empty spaces of your navbar?